### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -8,10 +8,10 @@ NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
 NLPModelsJuMP = "792afdf1-32c1-5681-94e0-d7bf7a5df49e"
 
 [compat]
-JuMP = "0.19, 0.20, 0.21"
-NLPModels = "0.12"
+JuMP = "0.21"
+NLPModels = "0.13"
 NLPModelsJuMP = "0.6"
```
Comment (abelsiqueira) [inline Project.toml:13]: There is no change in the NLPModelsJuMP version?

Comment (amontoison) [reply Project.toml:13]: No, maybe we will release the version 0.7 after https://github.com/JuliaSmoothOptimizers/NLPModelsJuMP.jl/pull/53.

Comment (amontoison) [reply Project.toml:13]: I wanted to add an example with a MathOptNLSModel of this package inside the documentation of NLPModelsJuMP and I encounter unsatisfiable requirements.

Comment (abelsiqueira) [reply Project.toml:13]: So do we need to drop support, or can we just add NLPModels 0.13?

Comment (amontoison) [reply Project.toml]: Let's drop support for NLPModels 0.12.x

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**